src/adapter.coffee

```diff
@@ -77,7 +77,7 @@ Rivets.public.adapters['.'] =
                 callbacks = map.callbacks
 
                 if callbacks[keypath]
-                  callback() for callback in callbacks[keypath].slice() when callback in callbacks[keypath]
```

Comment:
I think it was an oversight or remaining of a refactoring

We already know that the callback is in the array, since we just looped over it. There's no need to check again if is in the array. Looking at the JS generated code helps to understand.